docs/products/dragonfly/howto/eviction-policy-df.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Data eviction policy in Aiven for Dragonfly
+sidebar_label: Data eviction policy
+---
+
+Aiven for Dragonfly® optimizes cache memory management with a data eviction policy, ensuring efficient memory usage and optimal performance.
+
+## Understand data eviction policy
+
+Aiven for Dragonfly® uses a variation of the 2Q key eviction algorithm, which
+intelligently manages cache by deciding which data to retain or evict based on usage
+patterns. This system uses two distinct buffers to streamline cache memory management:
+
+- **Protected buffer:** Stores frequently accessed items, indicating their higher value
+  for caching due to repeated use.
+- **Probationary buffer:** Temporarily stores new or less frequently accessed items.
+  These items move to the protected buffer upon repeated access, indicating their growing
+  significance.
+
+This dual-buffer approach ensures resilience against fluctuating access patterns,
+maintaining cache efficiency without additional memory overhead per item.
+It's a key factor in Dragonfly's ability to keep valuable data accessible while managing
+less critical data efficiently.
+
+## Enable `cache_mode` via Aiven Console
+
+By default, `cache_mode` in Aiven for Dragonfly is disabled, providing basic caching.
```

Comment:
`cache_mode` is indeed disabled by default, but that means that no caching is used. So as soon as `maxmemory` is reached, writes will fail with an OOM exception.

Reply:
Updated text.